lib/sycamore/sycamore/tests/unit/connectors/opensearch/test_opensearch.py

```diff
@@ -420,7 +420,9 @@ def test_get_knn_query(self):
                     },
                     "index_name": "test_index",
                 },
-                "default": {"text_embedder": embedder},
+                "default": {
+                    "text_embedder": embedder,
```

Comment:
Think this came up because of the trailing comma after embedder. Could you remove it to minimize irrelevant diffs?

Reply:
black/ruff did this, not me

Reply:
Yes, I mean it did it because of the trailing comma.

Reply:
Our convention seems to be have trailing commas after each item? At least from what I see - there's probably a mix.

Reply:
I tried to remove the command and whitespace, it reverts to this in auto fix 🤷

Reply:
Unfortunate :/